ppr-ui/src/views/newMhrRegistration/MhrReviewConfirm.vue

```diff
@@ -44,10 +47,11 @@
         <AccountInfo
           v-if="accountInfo"
           class="mt-15"
-          title="Submtting Party for this Registration"
+          title="Submitting Party for this Registration"
           :tooltipContent="'The default Submitting Party is based on your BC Registries user account information. ' +
                             'This information can be updated within your account settings.'"
           :accountInfo="accountInfo"
+          desc = "Registration verification statement and decals will be mailed to this address."
```

Comment:
Small formatting: no spaces around = , and it could be placed under the `title` prop for consistency.